Packet++/header/TcpLayer.h

```diff
@@ -138,54 +144,111 @@ namespace pcpp
 		TCPOPT_Unknown =        255
 	};
 
+	/**
+	 * TCP options types
+	 */
+	enum class TcpOptionEnumType : uint8_t
+	{
+		/** Padding */
+		NOP =            1,
+		/** End of options */
+		EOL =            0,
+		/** Segment size negotiating */
+		MSS =          	 2,
+		/** Window scaling */
+		WINDOW =         3,
```

Comment:
let's change the items in `TcpOptionEnumType` to camel style, e.g. `WINDOW` -> `Window`. `SACK_PERM` -> `SackPerm`, to match the consistency of the rest of the codebase.